packages/liveblocks-react/rollup.config.js

```diff
@@ -1,20 +1,43 @@
-/**
- * This configuration is heavily inspired from https://github.com/pmndrs/zustand/blob/main/rollup.config.js
- */
-import path from "path";
-import resolve from "@rollup/plugin-node-resolve";
 import babelPlugin from "@rollup/plugin-babel";
-import typescript from "@rollup/plugin-typescript";
-import esbuild from "rollup-plugin-esbuild";
+import commandPlugin from "rollup-plugin-command";
```

Comment:
Please ignore all the noise caused here. If you've reviewed `rollup.config.js` in the client project, it's now 99.999% the same for the other projects. Only differences are the input file params — think `['index.ts', 'internal.ts']` vs `['index.tsx']`!